### PR TITLE
fix: content to upgrade DevLake

### PR DESCRIPTION
**Pull request by Startrekzky**
### ⚠️ &nbsp;&nbsp;Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have `npm run build` and `npm run serve` locally before submitting this PR
- [x] I have read through the [Contributing](https://devlake.apache.org/community/) Documentation

# Summary
Fix the content to upgrade DevLake

### Screenshots
image

image

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: versioned_docs/version-v0.16/GettingStarted/DockerComposeSetup.md [hunk 1]
```diff
@@ -12,26 +12,24 @@ sidebar_position: 1
 
 ## Launch DevLake
 
-Commands written `like this` are to be run in your terminal.
+Commands written `like this` are to be run in your terminal or pasted in the web browser.
 
-1. Download `docker-compose.yml` and `env.example` from the [latest release](https://github.com/apache/incubator-devlake/releases/tag/v0.16.1-beta1) into a folder.
-2. Rename `env.example` to `.env`. For Mac/Linux users, please run `mv env.example .env` in the terminal. This file contains the environment variables that the Devlake server will use. Additional ones can be found in the compose file(s).
+1. Download "docker-compose.yml" and "env.example" from the [latest release](https://github.com/apache/incubator-devlake/releases/tag/v0.16.1-beta1) into a folder.
```
Comment (klesh) [inline versioned_docs/version-v0.16/GettingStarted/DockerComposeSetup.md]: I suggest we keep the backquote as they are meant to be.

Comment (Startrekzky) [reply versioned_docs/version-v0.16/GettingStarted/DockerComposeSetup.md]: > I suggest we keep the backquote as they are meant to be.

But the original idea was to ONLY backquote `commands`. image

File: docs/GettingStarted/DockerComposeSetup.md [hunk 2]
```diff
@@ -12,26 +12,28 @@ sidebar_position: 1
...
-2. To view dashboards, click _View Dashboards_ button in the top left corner, or visit `localhost:3002` (username: `admin`, password: `admin`).
+   - "devlake" container takes a while to fully boot up. If `config-ui` complains about API being unreachable, please wait a few seconds and refresh the page.
+2. To view dashboards, click _View Dashboards_ button in the top left corner, or visit `localhost:3002` (username: "admin", password: "admin").
    - We use [Grafana](https://grafana.com/) to visualize the DevOps [data](/Overview/SupportedDataSources.md) and build dashboards.
    - For how to customize and provision dashboards, please see our [Grafana doc](../Configuration/Dashboards/GrafanaUserGuide.md).
 
 ## Upgrade to a newer version
 
-Support for database schema migration was introduced to DevLake in v0.10.0. From v0.10.0 onwards, users can upgrade their instance smoothly to a newer version. However, versions prior to v0.10.0 do not support upgrading to a newer version with a different database schema.
+1. Run `docker-compose down` to stop services and remove the volumes;
```
Comment (klesh) [inline docs/GettingStarted/DockerComposeSetup.md]: No remove volumes;  that would delete all data

**Review by klesh — APPROVED** [review]
LGTM

**Pull request merged**